alienv

```diff
@@ -253,22 +257,46 @@ case "$ACTION" in
     MODULES=$(normModules "${ARGS[@]}")
     collectModules
     existModules $MODULES
-    eval $($MODULECMD bash add $MODULES 2> >(grep -v "$IGNORE_ERR" >&2))
+    PS1DEV=
+    if [[ $DEVOPT == 1 ]];then
+      PS1DEV=" (dev)"
+      for MODULE in "$MODULES"; do
+        # echo "source $WORK_DIR/$ARCHITECTURE/$MODULE/etc/profile.d/init.sh"
+        eval $($MODULECMD bash add $MODULES 2> >(grep -v "$IGNORE_ERR" >&2))
+        source $WORK_DIR/$ARCHITECTURE/$MODULE/etc/profile.d/init.sh
+      done
+    else
+      eval $($MODULECMD bash add $MODULES 2> >(grep -v "$IGNORE_ERR" >&2))
+    fi
     [[ $UNAME == Darwin ]] && eval $(stripDyld)
     detectShell
     if [[ ! -z "$CLEAN_ENV" ]]; then
       case $MODULES_SHELL in
-        sh|bash) export PS1="[$MODULES]"' \w $> '   ;;
-        ksh)     export PS1="[$MODULES]"' $PWD $> ' ;;
-        zsh)     export PS1="[$MODULES]"' %~ %#> '  ;;
+        sh|bash) export PS1="[$MODULES]$PS1DEV"' \w $> '   ;;
+        ksh)     export PS1="[$MODULES]$PS1DEV"' $PWD $> ' ;;
+        zsh)     export PS1="[$MODULES]$PS1DEV"' %~ %#> '  ;;
       esac
     fi
     $MODULECMD bash list
-    printf "${ET}Use ${EM}alienv list${ET} to list loaded modules. " >&2
+    if [[ $DEVOPT == 1 ]];then
+      echo "*** Development env loaded for '$MODULES' ***"
+    else
+      printf "${ET}Use ${EM}alienv list${ET} to list loaded modules. " >&2
+    fi
     printf "Use ${EM}exit${ET} to exit this environment.${EZ}\n" >&2
     exec ${CLEAN_ENV:+env $SHELL_NORC_ENV} $MODULES_SHELL ${CLEAN_ENV:+$SHELL_NORC_PARAM} -i
   ;;
   printenv|load|unload)
+    if [[ $DEVOPT == 1 ]];then
+      echo "*** Please run following commands to load dev environment for '$MODULES' ***"
+      echo
+      echo "export WORK_DIR=$WORK_DIR"
+      for MODULE in "$MODULES"; do
```

Comment:
See above -- do not quote `$MODULES` here. If there are multiple modules, they'll be space-separated in this variable, and if you quote it, then bash won't iterate over the modules.